src/views/pages/login/Login.js

```diff
@@ -11,60 +11,71 @@ const Login = ({ authenticate }) => {
     const fetchData = async () => {
       const authorized = await isLoggedIn();
       return authenticate(authorized);
-    }
+    };
     fetchData();
   }, [authenticate]);
 
   return (
-    <div className="container">
-      <div className="container__item landing-page-container">
-        <div className="content__wrapper">
-          <header className="header">
+    <div className="container fluid">
+      <div className={styled["landing-page-container"]}>
+        <div className={styled.content__wrapper}>
+          <header className={styled.header}>
             <h1 className="d-flex justify-content-center display-1">BUNDLY</h1>
-            <ul className="social-container header__item">
-              <li className="social__icon social__icon--fb">
+            <ul
+              className={`${styled["social-container"]} ${styled.header__item}`}
+            >
+              <li
+                className={`${styled.social__icon} ${styled["social__icon--fb"]}`}
+              >
                 <img
                   src="https://image.flaticon.com/icons/svg/2111/2111425.svg"
                   alt="github"
                 />
               </li>
-              <li className="social__icon social__icon--dr">
+              <li
+                className={`${styled.social__icon} ${styled["social__icon--dr"]}`}
+              >
                 <img
                   src="https://image.flaticon.com/icons/svg/841/841568.svg"
                   alt="website"
                 />
               </li>
             </ul>
           </header>
-          <p className="coords">N 49° 16' 35.173" / W 0° 42' 11.30"</p>
-          <div className="ellipses-container">
-            <div className="greeting">
+          <p className={styled.coords}>S 75° 6' 0.027" / E 123° 19' 59.998"</p>
```

Comment:
Just curious on what these co-ordinates signify

Reply:
xD Wanted to signify it will be sent soon to Arctic Chamber